### ISSUE TITLE
мигает слайдер при обновлении страницы

### ISSUE THREAD

**Issue by kaznache** [nolimits4web/swiper#2666]
вот пример https://jsfiddle.net/mtonu8b0/1/
каждый раз при запуске сначала показывается первый слайд, а потом уже тот, который указан в initialSlide

это как-то решается?

**Comment by Craigy-** [comment]
По ссылке при запуске показывается Slide 2

**Comment by kaznache** [comment]
да, там второй слайд, но при перезапуске (Run) на доли секунды сначала виден первый слайд. на JSFiddle может и не очень заметно, но если перенести то же самое локально и многократно обновлять, то это мигание очень даже заметно

**Comment by kaznache** [comment]
была проверка на абсолютно пустом проекте. кроме одного слайдера из примера, ничего не было, соединение было отличное, но все равно мигает.
я понимаю, что первым идет Slide 1, а initialSlide это просто сдвиг, но если бы без этого мелькания при обновлении, было бы шикарно

получается, это никак не поправить?

**Comment by tv1ster** [comment]
Можно просто прятать слайдер, пока он не заинициализорвался.
https://jsfiddle.net/mtonu8b0/2/

**Comment by kaznache** [comment]
все равно мигает от пустоты ко второму слайду, но это гораздо лучше, чем с миганием первого слайдера

спасибо

**Comment by tv1ster** [comment]
Ну тут уже вы сами можете навесить какую-то анимацию, чтобы все происходило более плавно, либо запилить лоадинг.
